app/src/main/java/ar/com/wolox/android/example/ui/newsDetail/NewsDetailActivity.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.example.ui.newsDetail
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class NewsDetailActivity : WolmoActivity() {
+
+    companion object {
+        fun startNewsDetailActivity(context: Context, news: News) {
```

Comment:
Why this in a companion object? It shouldn't be here. You have to move it to NewsDetailFragment if needed